src/components/movie-items/movie-items.tsx

```diff
@@ -0,0 +1,47 @@
+import { MovieCard } from "components/movie-card/movie-card";
```

Comment:
I would prefer to rename file and items like `MockMovies` since you will upload real ones from server later